lib/airbrake/rack/user.rb

```diff
@@ -50,7 +50,12 @@ def as_json
       private
 
       def try_to_get(key)
-        String(@user.__send__(key)) if @user.respond_to?(key)
+        String(@user.__send__(key)) if can_call_method?(key)
```

Comment:
guys, this is not cool, i don't have neither the time nor patience to change the code up time and time again, until all of you like it. Discuss first internally and then give suggestions.